src/components/ItemEditorPage/CenterPanel/CenterPanel.tsx

```diff
@@ -43,8 +43,9 @@ export default class CenterPanel extends React.PureComponent<Props, State> {
       // The TP collections wouldn't have emotes soon, for this reason, we are fetching only standard collections to show in the Play Emote dropdown
       if (collection && !isTPCollection(collection)) {
         onFetchCollectionItems(collection.id)
-      } else {
-        onFetchOrphanItems(address!)
+      } else if (address) {
+        // TODO: Remove this call when there are no users with orphan items
+        onFetchOrphanItems(address)
```

Comment:
You can use the `hasUserOrphanItems` value to check if it's undefined to fetch the orphan items or not as well, what do you think about that?